### DISCUSSION TITLE
How import Components from Grafana UI (standalone package)

### DISCUSSION THREAD

**Discussion by chandra-96** [grafana/grafana#65690] [General]
I have built a custom plugin using grafana/ui (v9.1.2) but post 9.2.0 Grafana bundling has been changed, so once i have updated my packages to latest all my existing imports are failing
Can some help me with this ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
